### ISSUE TITLE
Allow to configure extensions for syntax highlighting

### ISSUE THREAD

**Issue by thiagodp** [alexkrechik/VSCucumberAutoComplete#191]
Useful to enable syntax highlighting for other file extensions that use a Gherkin-based syntax.

Something like this:
```js
...
  "cucumberautocomplete.extensions": [ ".feature", ".testcase" ]
...
```

**Comment by alexkrechik** [comment]
Files extensions are hardcoded, but I can add .testcase file extension in the future release

**Comment by alexkrechik** [comment]
Should be fixed in 2.12.0
